src/Microsoft.Health.Fhir.Core/Features/Conformance/ListedRestComponent.cs

```diff
@@ -8,21 +8,25 @@
 
 namespace Microsoft.Health.Fhir.Core.Features.Conformance
 {
-        public class ListedRestComponent
-        {
-            public string Documentation { get; set; }
+    public class ListedRestComponent
+    {
+        public string Documentation { get; set; }
 
-            public IList<RestfulCapabilityMode> Mode { get; set; }
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "This is a DTO-style class")]
```

Comment:
Should this be suppressed on the class level?

Reply:
Oh, for some reason I thought that would not work. Much nicer!